templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Are you sure you want delete {{ car.model }}?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button type="submit" value="Yes, delete" class="btn btn-danger">Yes, delete</button>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist